front/src/NotificationWindow.js

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import styled from "styled-components";
+import faTimes from "@fortawesome/fontawesome-free-solid/faTimes";
+import FontAwesomeIcon from "@fortawesome/react-fontawesome";
+
+const Window = styled.div`
+  background-color : ${props => props.backgroundColor}
+  border-style: solid;
+  border-size: 4px;
+  width: 90%;
+  position: relative;
+  margin-left:auto;
+  margin-right:auto;
+`;
+
+const NotificationHeader = styled.p`
+  text-align: center;
+  font-size: 5vmin;
```

Comment:
why not `rem`s?

Reply:
Tom used the `vmin` tag, but I'll change them to `rem` instead. I'll try to preserve layout and behaviour upon viewport changes.

Reply:
Using `font-size: rem`, the font-size is no longer responsive. If you have any idea how to keep the font-size responsive but with `rem` instead, I suggest you fix this because I do not have a clue how to do this.

Reply:
just leave `rem`s